Source/diablo.cpp

```diff
@@ -251,8 +251,10 @@ int APIENTRY WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLi
 #endif
 	ghInst = hInst;
 
+#ifndef __ANDROID__
 	if (RestrictedTest())

```

Comment:
What specifically is it that fails on Andoid about RestrictedTest?

Reply:
Hey @AJenbo ,

I will check this weekend but last time I was compiling the code it was complaining we're not on Windows95 or something like that  :D 
Probably easy thing to fix, just commenting that part was faster :)

Reply:
Thanks that would be helpful

Reply:
Good news after investigating things a big we decided to drop the `RestrictedTest`. For one thing it wasn't really relevant for what we are doing and it wasn't even present in the official Mac OS 9 :)

Reply:
Diablo was made in a time period where everything was automatically given full access to the system, so it was not a problem. 1.08 came out shortly after Windows 2000 to address the fact that everything runs in user-mode, and also moved multi saves out of the windows folder. This is even more relevant nowadays since windows vista and 10 restricted access even further. X doesn't write to a restricted folder tho.